src/components/Similarity/SimilarFilesSidebar.tsx

```diff
@@ -83,17 +85,42 @@ const SimilarEntriesComponent: React.FC<SimilarEntriesComponentProps> = ({
     };
   }, [filePath]);
 
+  const updateSimilarEntries = async () => {
+    const searchResults = await performSearch(filePath);
+    if (searchResults.length > 0) {
+      setSimilarEntries(searchResults);
+    }
+  };
+
   return (
-    <div className="h-full overflow-y-auto overflow-x-hidden mt-0 border-l-[0.1px] border-t-0 border-b-0 border-r-0 border-gray-600 border-solid">
-      {similarEntries.length > 0 && (
-        <div className="flex items-center justify-center bg-gray-800 mt-0 mb-0 p-0">
-          <PiGraph className="text-white mt-1" />
+    <div
+      className={`h-below-titlebar ${
+        similarEntries.length > 0 ? "overflow-y-auto" : "overflow-y-hidden"
+      } overflow-x-hidden mt-0 border-l-[0.1px] border-t-0 border-b-0 border-r-0 border-gray-600 border-solid`}
+    >
+      {" "}
+      {/* {similarEntries.length > 0 && ( */}
+      <div className="flex items-center bg-gray-800 p-0">
+        {/* Invisible Spacer */}
+        <div className="flex-1"></div>
 
-          <p className="text-gray-200 text-sm pl-1 mb-0  mt-1 pt-0 pb-0">
-            Related Notes
-          </p>
+        {/* Centered content: PiGraph icon and Related Notes text */}
+        <div className="flex items-center justify-center px-4">
+          <PiGraph className="text-gray-300 mt-1" />
+          <p className="text-gray-300 text-sm pl-1 mb-0 mt-1">Related Notes</p>
         </div>
-      )}
+
+        <div
+          className="flex-1 flex justify-end pr-3 pt-1 cursor-pointer"
+          onClick={() => {
+            setUserHitRefresh(true);
+            setSimilarEntries([]); // simulate refresh
+            updateSimilarEntries();
```

Comment:
Can we remove setSimilarEntries? and just [set similar entries regardless](https://github.com/reorproject/reor/pull/108/files#r1512161836) if we hit refresh?

Reply:
I just added this because it simulates a refresh, otherwise sometimes it looks like nothing happens if you hit refresh